data/json/recipes/recipe_vehicle.json

```diff
@@ -56,6 +56,19 @@
       [ [ "rope_6", 2 ] ]
     ]
   },
+  {
+    "result": "tow_cable",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_VEHICLE",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": 120000,
+    "autolearn": true,
+    "qualities": [ { "id": "HAMMER", "level": 1 } ],
+    "using": [ [ "welding_standard", 2 ] ],
+    "components": [ [ [ "duct_tape", 50 ] ], [ [ "grip_hook", 1 ] ], [ [ "chain", 2 ] ] ]
```

Comment:
add some "cable" (ie, copper wire for the electrical hookups) and enough chain to get up to 24 kgs of total weight.

time should be 8 hours or so.

Reply:
The tow cable has electrical hookups?